### PR TITLE
OJS3 Bahasa Indonesia Translation (January 19, 2019)

### PR DESCRIPTION
**Pull request by oichidan**
Hello,
My name is Aldi. I am the website administrator of http://ijsed.ap3si.org/, a journal website using OJS3. My journal website is written in Bahasa Indonesia. When I first used the provided Bahasa Indonesia translation, I noticed many lines are missing. I've decided to use my spare time to review and update the existing translation, matching them with the English translation, adding necessary XML entries to make sure the Bahasa Indonesia translation keeps up with the English version.

Please review the changes I've done today from the pull request. As a general view, I've done the following:
- Edited and updated `locale\id_ID\default.xml`
- Added `locale\id_ID\api.xml`
- Edited and updated `locale\id_ID\admin.xml`
- Edited and updated `locale\id_ID\editor.xml`
- Edited and updated `locale\id_ID\emailTemplates.xml`

I will continue editing the remaining files of the Bahasa Indonesia translation in my spare time. If there's anything else I can be of help, please let me know.

Thanks in advance.

Warm regards,

Aldi
+628112163825
http://t.me/oichidan

### CODE REVIEW

**Comment by asmecher** [comment]
Thanks for contributing, @oichidan! Tagging @mtub, who manages translations.

**Comment by oichidan** [comment]
You're welcome. I'm currently working on some other files and I'll create another pull request once everything's all set up.

Let me know if I can be of further help.

**Comment by mtub** [comment]
Thanks, @oichidan! I will start reviewing it now.

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ node_modules
 js/build.js
 js/hot-updates
 styles/build.css
+.vs/
```
Comment (mtub) [inline .gitignore]: please don't add new stuff here.

File: locale/id_ID/api.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  * locale/id_ID/api.xml
+  *
+  * Copyright (c) 2014-2018 Simon Fraser University
+  * Copyright (c) 2000-2018 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the REST API.
+  -->
+
+<locale name="en_US" full_name="U.S. English">
```
Comment (mtub) [inline locale/id_ID/api.xml]: this should be `<locale name="id_ID" full_name="Bahasa Indonesia">`

**Pull request closed without merging**